docs/record-encryption-quickstart/index.adoc

```diff
@@ -0,0 +1,254 @@
+Use Helm to install either HashiCorp Vault or AWS LocalStack.
+
+==== HashiCorp Vault
+
+[source,shell]
+----
+helm repo add --force-update hashicorp https://helm.releases.hashicorp.com
+helm upgrade --install vault hashicorp/vault --namespace vault --create-namespace --version 0.30.1 --set server.dev.enabled=true,server.dev.devRootToken=myroottoken --wait
+----
+
+You also need to enable the Transit Secrets Engine in Vault.
+
+[source,shell]
+----
+kubectl exec -n vault statefulsets/vault -- vault secrets enable transit
+----
+
+==== AWS Local Stack
+
+[source,shell]
+----
+helm repo add --force-update localstack-repo https://helm.localstack.cloud
+helm upgrade --install localstack localstack-repo/localstack --namespace localstack --create-namespace --version 0.6.24 --set service.type=ClusterIP --wait
+----
+
+=== Install the Strimzi Operator
+
+We are going to use Strimzi to deploy the Kafka Cluster that will be proxied.
+Let's install Strimzi next:
+
+[source,shell]
+----
+helm repo add --force-update strimzi https://strimzi.io/charts/
+helm upgrade --install strimzi-operator strimzi/strimzi-kafka-operator --namespace kafka --create-namespace --version 0.47.0
+----
+
+=== Install the Kroxylicious Operator
+
+First, create a temporary directory for some working files.  We use a directory called `ko-install` beneath `/tmp`, but you can use any name and location you like.
+
+[source,shell]
+----
+mkdir /tmp/ko-install
+cd /tmp/ko-install
+----
+
+Now let's install the Kroxylicious Operator
+
+[source,shell]
+----
+curl --location https://github.com/kroxylicious/kroxylicious/releases/download/v0.13.0/kroxylicious-operator-0.13.0.zip --output kroxylicious-operator.zip
+unzip kroxylicious-operator.zip
+kubectl apply -f install
+kubectl wait --namespace kroxylicious-operator deployment/kroxylicious-operator --for=condition=Available=True --timeout=300s
+----
+
+== Deploy the Kafka Cluster
+
+Now we need a Kafka Cluster.
+This is the Kafka Cluster that will be proxied.
+We'll just use one from the Strimzi Quickstart.
+It will create a cluster in the `kafka` namespace
+
+[source,shell]
+----
+kubectl apply -n kafka -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/refs/tags/0.47.0/examples/kafka/kafka-single-node.yaml
+kubectl wait -n kafka kafka/my-cluster --for=condition=Ready --timeout=300s
+----
+
+We'll need the boostrap server address of the cluster later, so let's assign a variable containing it now.
+
+[source,shell]
+----
+DIRECT_CLUSTER_BOOTSTRAP_SERVERS=$(kubectl get -n kafka kafka my-cluster -o=jsonpath='{.status.listeners[0].bootstrapServers}')
+----
+
+
+== Deploy the Proxy with Record Encryption
+
+Next, we'll deploy a Kroxylicious proxy instance using the record encryption example included in the install zip you download above.
+
+It will create an instance of Kroxylicious that will proxy the kafka cluster you created a above, and configure it to
```

Comment:
```suggestion
Next, we'll deploy a Kroxylicious proxy instance using the record encryption example included in the install zip you download above.

It will create an instance of Kroxylicious that will proxy the kafka cluster you created above, and configure it to
```